plugins/tracker-resources/src/components/CreateIssue.svelte

```diff
@@ -395,9 +400,22 @@
     }
   }
 
+  const projectPreferences = createQuery()
+  let preferences: ProjectTargetPreference[] = []
+  $: projectPreferences.query(tracker.class.ProjectTargetPreference, {}, (res) => {
+    preferences = res
+  })
+  $: spacePreferences = preferences.find((it) => it.attachedTo === _space)
+
   async function createIssue (): Promise<void> {
     const _id: Ref<Issue> = generateId()
-    if (!canSave || object.status === undefined || _space === undefined || kind === undefined) {
+    if (
+      !canSave ||
+      object.status === undefined ||
```

Comment:
these conditions seem to repeat what we have in `canSave`, why do we need this?

Reply:
It is required to make typescript happy.